arm-network/2016-09-01/swagger/routeFilter.json

```diff
@@ -0,0 +1,849 @@
+            "schema": {
+              "$ref": "#/definitions/RouteFitlerRule"
+            },
+            "description": "Parameters supplied to the create or update route filter rule operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Update successful. The operation returns the resulting Route Filter Rule resource.",
+            "schema": {
+              "$ref": "#/definitions/RouteFitlerRule"
+            }
+          },
+          "201": {
+            "description": "Create successful. The operation returns the resulting Route Filter Rule resource.",
+            "schema": {
+              "$ref": "#/definitions/RouteFitlerRule"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "patch": {
+        "tags": [
+          "RouteFilterRules"
+        ],
+        "operationId": "RouteFilterRules_Update",
+        "description": "Updates a route in the specified route filter.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "routeFilterName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the route filter."
+          },
+          {
+            "name": "ruleName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the route filter rule."
+          },
+          {
+            "name": "routeFilterRuleParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RouteFitlerRule"
+            },
+            "description": "Parameters supplied to the update route filter rule operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Update successful. The operation returns the resulting Route Filter Rule resource.",
+            "schema": {
+              "$ref": "#/definitions/RouteFitlerRule"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/routeFilters/{routeFilterName}/routeFilterRules": {
+      "get": {
+        "tags": [
+          "RouteFilterRules"
+        ],
+        "operationId": "RouteFilterRules_List",
```

Comment:
Standard naming Convention for child resources:
`RouteFilters_ListByRouteFilter`